src/theory/quantifiers/ceg_instantiator.cpp

```diff
@@ -294,9 +294,13 @@ bool CegInstantiator::doAddInstantiation( SolvedForm& sf, unsigned i, unsigned e
               Node lit = ita->second[j];
               if( std::find( lits.begin(), lits.end(), lit )==lits.end() ){
                 lits.push_back( lit );
-                if( vinst->processAssertion( this, sf, pv, lit, effort ) ){
-                  return true;
-                }
+                //if( vinst->hasProcessAssertion( this, sf, pv, lit, effort ) ){
+                  // apply substitutions check if eligible and contains pv
+                  // TODO
```

Comment:
Could you link each TODO to a Github issue, so that they don't get lost?

Reply:
See general comment [1] below.